build/script/steamcmd

```diff
@@ -1,7 +1,3 @@
 #!/usr/bin/env bash
 
-if [ `whoami` != "$STEAM_USER" ]; then
-	exec gosu $STEAM_USER $0 "$@"
-fi
-
```

Comment:
I will give it a try and if something doesn't work out we will change it before the final release. Thanks for your feedback and expertise on this!